services/nublado2/values-bleed.yaml

```diff
@@ -0,0 +1,49 @@
+nublado2:
+  jupyterhub:
+    debug:
+      enabled: true
+
+    hub:
+      image:
+        name: lsstsqre/nublado2
+        tag: "1.0.0"
+
+    singleuser:
+      storage:
+        type: none
+
+    auth:
+      state:
+        cryptoKey: "080c6d0eb8d9f84ef5f0860bea71dba6ab2e1be712c7bc8c2eaf92321c9fd968"
+
+    proxy:
+      secretToken: "3854245f5b48412ee9b7a2c985da6acdc146f80cc4f10eb175f0efa24530399b"
+      service:
+        type: ClusterIP
+
+    ingress:
+      hosts: ["bleed.lsst.codes"]
+      annotations:
+        nginx.ingress.kubernetes.io/auth-signin: "https://bleed.lsst.codes:31337/login"
+        nginx.ingress.kubernetes.io/auth-url: "http://gafaelfawr-service.gafaelfawr.svc.cluster.local:8080/auth?scope=exec:notebook"
```

Comment:
The `auth-url` must be the external URL, not the internal one.

Reply:
Ack, ignore this, this is fine. I had completely forgotten our previous conversation about this and that you'd tested it and it works fine. (In fact, I probably should stop exposing that route to the Internet since there's no reason for people to use it directly.) Same note as below about `.svc.cluster.local`, though.